cmd/ig/containers/containers.go

```diff
@@ -86,6 +86,10 @@ func NewListContainersCmd() *cobra.Command {
 			col.Visible = true
 			col, _ = cols.GetColumn("runtime.containerImageName")
 			col.Visible = true
+			// hide the k8s column
+			for _, c := range cols.GetColumnMap(columns.WithTag("kubernetes")) {
+				c.Visible = false
+			}
```

Comment:
Please synch this with what you did in `pkg/container-collection/containers.go`. Maybe you could even unify it in a function to avoid having to update it in both places each time.

Reply:
Done